### ISSUE TITLE
ARM image?

### ISSUE THREAD

**Issue by Roming22** [hwdsl2/docker-ipsec-vpn-server#164]
Hi

Is there any chance you could provide an image for ARM (e.g. RaspberryPi3 and 4)?
Your project works fine on that architecture, but it would just be more convenient to not have to check for updates on your repo and build new images for projects that uses this project as a base image.

Regards

**Comment by shaomeng** [comment]
@Roming22 Just FYI, I built the docker image for my raspberry pi following steps described [here](https://github.com/hwdsl2/docker-ipsec-vpn-server#build-from-source-code), and it's quite easy and works well!

**Comment by Roming22** [comment]
@shaomeng Yes, as my post said the project works fine on ARM. It is just that I have another image based on that project, and it is a pain having to make sure that I don't miss updates.
If there was an ARM base image, I could just have a `FROM hwdsl2/docker-ipsec-vpn-server` and I'd be guaranteed to use an up to date version.

**Comment by hwdsl2** [comment]
@Roming22 Thank you for your suggestion. Currently, Docker Hub automated builds do not support building multi-arch images [1]. For this reason, please continue to use the workaround in [2] for now.

[1] https://github.com/moby/moby/issues/36552
[2] https://github.com/hwdsl2/docker-ipsec-vpn-server#use-on-raspberry-pis

**Comment by hwdsl2** [comment]
Fixed in de29744. Multi-arch images are now available:
```
linux/amd64, linux/386, linux/arm64, linux/arm/v7 and linux/arm/v6
```

**Comment by Roming22** [comment]
Thanks!
